docs/dev/release-process.md

```diff
@@ -28,5 +28,27 @@ While the `[do not deploy]` can be anywhere in the title, it is recommended to p
 The deployment pipeline will analyse the message of the merge commit (which will be contain the PR title) and make a decision based on it.
 
 ## Releasing to production
-The process to promote a build from `stage` to `production` is currently manually initiated by SRE.
-[This ticket](https://mozilla-hub.atlassian.net/browse/CONSVC-1566) (requires LDAP) deals with automating the process.
+Developers with write access to the Merino repository can initiate a deployment to production after a Pull-Request on the Merino GitHub repository is merged to the `main` branch. This can be done by:
```

Comment:
Just for clarification, are `developers` referring to the same individual(s) who authored and merged the pull request, or do we need someone else to operate this including the approval process mentioned below? I think it should be the former but just wanted to double check.

Reply:
I was intentionally generic in this statement, because anyone could trigger, as long as they have write access to the Merino repository.

Ideally speaking, I would expect the individual who authored the PR to do that, but we don't strictly enforce this. The reason why I would expect that is that authors know the code and know what to watch out for when things are hitting production. If the author touched search providers in a subtle way and the plots show anomalies with search queries, then they might be better suited to identify the problem.

In general, we want developers / authors to feel the _responsibility_ of deploying to production, own that deployment and shepherd it to conclusion.

Reply:
There's a lot of helpful context in your response, @Dexterp37. I would love this to be captured in the documentation.